Utilities/DataLoading/EdFi.LoadTools/BulkLoadClient/Application/BulkLoadClientConfiguration.cs

```diff
@@ -165,7 +167,8 @@ public static BulkLoadClientConfiguration Create(IConfiguration configuration)
                 XsdMetadataUrl = configuration.GetValue<string>("OdsApi:XsdMetadataUrl"),
                 Extension = configuration.GetValue<string>("OdsApi:Extension"),
                 ApiMode =  apiMode,
-                ForceMetadata = configuration.GetValue<bool>("ForceMetadata")
+                ForceMetadata = configuration.GetValue<bool>("ForceMetadata"),
+                AdminAppSchemaAlreadyDownloaded = configuration.GetValue<bool>("AdminAppSchemaAlreadyDownloaded")
```

Comment:
@CSR2017 Why couldn't we use ForceMetadata configuration here? I think that should guide the download instead of adding a new configuration. Similar process is done for json metadata https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/main/Utilities/DataLoading/EdFi.LoadTools/ApiClient/SwaggerMetadataRetriever.cs#L41 cc @jamessmckay

Reply:
I think one of problems here is that the downloader is pretty aggressive. Even if you provide it with an existing folder of schema files to work off of, if it finds the xsd metadata url it will redownload the files even when ForceMetdata is false.

I was trying to work towards a solution where if you explicitly provide the folder and it is not empty it will attempt to use that as its schema source.

Reply:
> I think one of problems here is that the downloader is pretty aggressive. Even if you provide it with an existing folder of schema files to work off of, if it finds the xsd metadata url it will redownload the files even when ForceMetdata is false.
> 
> I was trying to work towards a solution where if you explicitly provide the folder and it is not empty it will attempt to use that as its schema source.

@tbmeador-dlp is that change coming in ODS-4858?

Reply:
@vimayya yes, I was hoping to include it with that ticket

Reply:
@vimayya @tbmeador-dlp 
Updated the code based on ODS-4858 changes. Need review and merge after merging https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/pull/298/files